### ISSUE TITLE
Error while setup models

### ISSUE THREAD

**Issue by dayue1984** [matiasdelellis/facerecognition#548]
In FileService.php line 333:
Curl error: HTTP/2 stream 1 was not closed cleanly before end of the underlying stream

**Comment by guystreeter** [comment]
Please provide the your facerecognition app version and Nextcloud version, the exact command you entered, and all of the output from it.

**Comment by Ling128** [comment]
same issues.
facerecognition app version:  0.9.1
Nextcloud version: 23.0.0
command: php occ face:setup -m 1
output: 
![image](https://user-images.githubusercontent.com/9321969/147069995-edf70eac-dcf7-41f7-b6b0-683a5161e8e6.png)

**Comment by guystreeter** [comment]
The error happens in the installation while using PHP's builtin "curl" to download the model data from a repository on github.

It's definitely a problem in the curl implementation. I've seen some reports of the "not closed cleanly" error happening on very slow connections. Perhaps github is throttling the download.

I started up a clean nextcloud install and tried the command without failure, twice. I also manually used the curl command-line to download the data files to my local machine, and that worked. I don't know what else to do diagnose this. Maybe @matiasdelellis has some suggestions.
